graphene_django/views.py

```diff
@@ -103,16 +103,15 @@ def __init__(
     ):
         if not schema:
             schema = graphene_settings.SCHEMA
+        self.schema = schema or self.schema
 
         if middleware is None:
             middleware = graphene_settings.MIDDLEWARE
+        if isinstance(middleware, MiddlewareManager):
```

Comment:
Let me know if I should've kept the check for `middleware is not None` here - my thought process was that we already check above if it's none, so probably we don't need to check if now it is _not_ none.

Unless `graphene_settings.MIDDLEWARE` can be `None` - then this is needed and I can re-add it back in.